packages/animations/pubspec.yaml

```diff
@@ -16,3 +16,15 @@ dev_dependencies:
   flutter_test:
     sdk: flutter
   vector_math: ^2.1.0
+
+screenshots:
+  - description: 'Thumbnail of a container transform animation.'
```

Comment:
I didn't see it in the example package, but do these descriptions show up somewhere on pub?

Reply:
No, they don't.